docformatter.py

```diff
@@ -631,23 +631,37 @@ def _get_encoding():
     return locale.getpreferredencoding() or sys.getdefaultencoding()
 
 
+def find_py_files(sources, recursive):
+    """Find Python source files.
+
+    Parameters:
+        - sources: iterable with paths as strings.
+        - recursive: drill down directories if True.
+
+    Return: yields paths to found files.
+    """
+    def not_hidden(name):
+        """Return True if file 'name' isn't .hidden."""
+        return not name.startswith('.')
+
+    for name in sorted(sources):
+        if recursive and os.path.isdir(name):
+            for root, dirs, children in os.walk(unicode(name)):
+                dirs[:] = sorted(filter(not_hidden, dirs))
+                for file in sorted(filter(not_hidden, children)):
```

Comment:
Can you rename `file` to `filename` (just to avoid collision with the Python 2.7 `file` class)?

Reply:
FWIW In Python 2 it’s recommended to ignore `file` and use exclusively `open` (and for instance checks, use capability checks or try/except), and in Python 3 there is no built-in anymore, which makes `file` available as the most natural way to name an object that is an open file (instead of `fp`, `fileobj`, `file_` and others).

But for objects that are filenames rather than file objects, `filename` is indeed better!

Reply:
Sure, thanks for notice.